<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res.qrc">
    <normaloff>:/ico/qgis.ico</normaloff>:/ico/qgis.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="mapcanvasWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVector_Layers">
    <property name="title">
     <string>Vector</string>
    </property>
    <widget class="QMenu" name="menuGDAL">
     <property name="title">
      <string>GDAL</string>
     </property>
     <addaction name="actionShapefile"/>
     <addaction name="actiondxf"/>
     <addaction name="actiongeopackage"/>
    </widget>
    <addaction name="actionQgisInterface"/>
    <addaction name="menuGDAL"/>
    <addaction name="actionpostgis"/>
    <addaction name="actioncsv"/>
    <addaction name="actiongpx"/>
    <addaction name="actionSpatiaLite"/>
    <addaction name="actionMySQL"/>
    <addaction name="actionWFS"/>
   </widget>
   <widget class="QMenu" name="menuRaster">
    <property name="title">
     <string>Raster</string>
    </property>
    <addaction name="actionTif"/>
    <addaction name="actionGeopackage"/>
    <addaction name="actionPostGIS"/>
    <addaction name="actionWCS"/>
   </widget>
   <addaction name="menuVector_Layers"/>
   <addaction name="menuRaster"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="layerTreeDockWidget"/>
  </widget>
  <action name="actionQgisInterface">
   <property name="text">
    <string>QgisInterface</string>
   </property>
  </action>
  <action name="actionpostgis">
   <property name="text">
    <string>PostGIS</string>
   </property>
  </action>
  <action name="actioncsv">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actiongpx">
   <property name="text">
    <string>GPX</string>
   </property>
  </action>
  <action name="actionSpatiaLite">
   <property name="text">
    <string>SpatiaLite</string>
   </property>
  </action>
  <action name="actionShapefile">
   <property name="text">
    <string>shapefile</string>
   </property>
  </action>
  <action name="actiondxf">
   <property name="text">
    <string>dxf</string>
   </property>
  </action>
  <action name="actiongeopackage">
   <property name="text">
    <string>geopackage</string>
   </property>
  </action>
  <action name="actionMySQL">
   <property name="text">
    <string>MySQL</string>
   </property>
  </action>
  <action name="actionWFS">
   <property name="text">
    <string>WFS</string>
   </property>
  </action>
  <action name="actionTif">
   <property name="text">
    <string>Tiff</string>
   </property>
  </action>
  <action name="actionGeopackage">
   <property name="text">
    <string>Geopackage</string>
   </property>
  </action>
  <action name="actionPostGIS">
   <property name="text">
    <string>PostGIS</string>
   </property>
  </action>
  <action name="actionWCS">
   <property name="text">
    <string>WCS</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
